arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -680,6 +680,48 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/virtualNetworks/{virtualNetworkName}/usages": {
+      "get": {
+        "operationId": "VirtualNetworks_GetUsage",
```

Comment:
This should be named as `_listUsage` as the model returns more than one values like array of values?